packages/api/src/event/medical/patient-event.ts

```diff
@@ -1,7 +1,12 @@
 import EventEmitter from "events";
 import { Patient } from "@metriport/core/domain/patient";
+import { MAPIWebhookType } from "../../domain/webhook";
 
 export type PatientEvent = Pick<Patient, "id" | "cxId">;
+export type CanvasIntegrationEvent = PatientEvent & {
```

Comment:
ditto: let's not call it Canvas event, this should be something like DocumentWebhookEvent or something more generic.